docs/admin/troubleshooting_network.rst

```diff
@@ -2,3 +2,218 @@ Troubleshooting network issues
+Common causes for lost connections include fully or partly unplugged network cables,
+lost power to networking equipment, and ISP service outages. When you see a lost
+connection notification, it is most likely due to one of these causes.
+
+If the network manager shows that you are connected to the Internet, you can
+verify whether your connection is working by opening a terminal in ``sys-net``:
+
+**[SCREENSHOT: Q widget with VM list and "Run terminal" expanded]**
+
+1. Click the "Q" icon in the in the system tray (top right area).
+2. A list of running VMs should appear. Select ``sys-net`` from the list, and
+   click **Run Terminal**.
+3. In the terminal window, type the command ``ping google.com``.
+
+You should see a sequence of lines starting with ``64 bytes from`` and ending with
+the number of milliseconds it took to complete the request. If you do not see
+similar output, your network access may be misconfigured, or the Internet may be
+wholly or partially unreachable.
+
+If you have verified that you are able to connect to the Internet using
+``sys-net``, but you are experiencing other connectivity issues, move on to the
+next step.
+
+.. important::
+
+   Not all VMs in Qubes OS have Internet access. For example, opening the Qubes
+   menu (top left) and clicking **Terminal Emulator** opens a ``dom0`` terminal
+   without Internet access. See our :ref:`networking architecture <Networking Architecture>`
+   overview for additional background.
+
+Step 2: Troubleshooting login issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Issues logging in may not be network-related. If you are experiencing
+connectivity issues before or after logging in, you can skip ahead to the next section.
+
+Make sure that your username, passphrase, and two-factor code are correct.
+
+.. important::
+
+   After a failed login, wait for a new two-factor code from your app before
+   trying again.
+
+You can reveal the passphrase by clicking the "eye" icon next to it in the login
+dialog (ensure you are in a fully private setting before doing so). Check for
+extra characters at the beginning at the end, or subtle differences like
+capitalization.
+
+If you use the two-factor app on your phone for other websites and services,
+make sure that you have selected the correct user account.
+
+If you have access to a Tails-based *Journalist Workstation*, verify whether you
+can access SecureDrop from Tails.
+
+If you are certain that your credentials are correct but you are unable to log
+in, proceed to the next step.
+
+Step 3: Verify that all required VMs are running
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The following VMs must be running for all actions requiring network connectivity
+to work (e.g., logging in, checking for messages, downloading documents, replying
+to sources, starring sources, deleting sources):
+
+- ``sd-app``
+- ``sd-gpg``
+- ``sd-log``
+- ``sd-proxy``
+- ``sd-whonix``
+- ``sys-firewall``
+- ``sys-net``
+- ``sys-whonix`` (during updates)
```

Comment:
Great list. The autostart functionality is sufficient for most use cases, but we should probably be _enforcing_ these VMs as running in the project code. Will file an issue to track post-pilot. Regardless, the list here should remain as useful debugging info.